### ISSUE TITLE
TinyMCE menu position not recalculated on scroll

### ISSUE THREAD

**Issue by frapell** [plone/plone.app.mosaic#369]
This is related to #301 

When you open a tinymce drop down for the richtext tile, and you scroll down so the toolbar position is fixed on top, opened menues position are not recalculated.

Here's a video showing the issue: https://drive.google.com/open?id=0B94hTF3v42BteUliQjlHdVZTdms

This is happening in latest master

**Comment by frapell** [comment]
@datakurre Also, check http://fiddle.tinymce.com/7Sfaab/1 is mosaic maybe preventing some event to be fired when going from float to fixed?

**Comment by datakurre** [comment]
@frapell I don't know. I haven't looked how Nathan implemented that stickyness. It could also be CSS related issue, because calculating vertical stuff is tricky. Already #301 was hard to discover :/

**Comment by frapell** [comment]
yeah, it took me quite some time to find... I notice how the "top" property for the menu change when switches from float to sticky on default tinymce (As in the fiddle link I posted), but it doesn't in Plone, so I thought there might be an event that is not firing in Plone...

I'll keep digging.

@vangheem thoughts?

**Comment by datakurre** [comment]
@frapell It's interesting that in fiddle TinyMCE behaves well out of the box, but in Mosaic Nathan had to add manual stickyness handling. That might be why "top" is missing.

https://github.com/plone/plone.app.mosaic/blob/master/src/plone/app/mosaic/browser/static/js/mosaic.tile.js#L28
